skimage/transform/_warps.py

```diff
@@ -325,6 +354,162 @@ def rotate(image, angle, resize=False, center=None, order=1, mode='constant',
                 mode=mode, cval=cval, clip=clip, preserve_range=preserve_range)
 
 
+def downsize(image, output_shape, sigma=None, order=1, mode="reflect", cval=0,
```

Comment:
Should this not become the default for resize and rescale?

Reply:
Not sure I understand the question correctly. Are you referring to all parameters or a specific one?

Reply:
Sorry, I meant the Gaussian blurring should eventually be the default (I think it's the "correct" way). As such, would it not make sense to have a binary flag added to the existing downsampling functions, instead of adding another two with vaguely similar names?

Reply:
Makes sense. This way, we could also smooth the image after interpolation when upsampling. Not sure about the recent policies in scikit-image but deprecation path is still with setting a parameter to None and warning the user?

Reply:
Yes, still: https://github.com/scikit-image/scikit-image/blob/master/CONTRIBUTING.txt#L383

Glad to have you back for a bit :)